bluebrain/repo-bluebrain/packages/py-bluepy-configfile/package.py

```diff
@@ -13,6 +13,7 @@ class PyBluepyConfigfile(PythonPackage):
     git      = "git@bbpgitlab.epfl.ch:nse/bluepy-configfile.git"
 
     version('develop', branch='main')
+    version('0.1.18', tag='bluepy-configfile-v0.1.18')
```

Comment:
@matz-e @mgeplf do you know why this still doesn't work? The tag was created 2 days ago. 

* add0aee - (tag: bluepy-configfile-v0.1.18, origin/main, origin/HEAD) Version 0.1.18 (2 days ago) <Mike Gevaert>
* 2df43f7 - [BLPY-278]: add support for Conductance stimuli (2 days ago) <Michael Gevaert>

Reply:
I updated py minis validation's bglibpy dependency to fix it. 

`depends_on('py-bglibpy@4.7.2:4.999', type='run')`

Reply:
It again fails.
That's the error message. I do not see any conflict between those constraints.

```
  imposed_constraint(4468,"version_satisfies","py-bluepy-configfile","0.1.18:")
  imposed_constraint(5217,"version_satisfies","py-bluepy-configfile","0.1.10:0.999")
```

Reply:
Does `"py-bluepy-configfile","0.1.18:"` include or exclude version `0.1.18`?

Reply:
Can one not add 2 updates (one for bglibpy one for bluepy-configfile) within one change?